### DISCUSSION TITLE
Github Education

### DISCUSSION THREAD

**Discussion by 4RKO0** [community/community#144264] [GitHub Education]
### Hi everyone,

this can be considered a bug because I was using student developer pack for my previous github account. i couldn't log in to my account because the two-factor application i was using turned off pc support and i had to close my account. when i opened a new account, i transferred my e-mail information to my new account, but i can't use the github copilot feature i was using because it says that it is already activated for this mail, but i can't use it. can anyone help or have information about this issue?

**Comment by davevad93** [comment]
Hi @4RKO0 ,

GItHub Education benefits are tied to the academic email used to submit the application, in your case the benefits are still linked to the email verified with your former profile, also it's not possible to transfer academic benefits from one account to another.

You could contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue, however I doubt anything can be done but try anyway.
